packages/transport/src/modules/relationships/local/PeerDeletionInfo.ts

```diff
@@ -20,6 +23,10 @@ export class PeerDeletionInfo extends Serializable implements IPeerDeletionInfo
     })
     public deletionStatus: PeerDeletionStatus;
 
+    @serialize()
+    @validate({ nullable: true })
+    public deletionDate?: CoreDate;
```

Comment:
This should not be nullable and instead default to now in 14 Days when the contact is to be deleted or now if the contact is already deleted.

Reply:
> and instead default to now in 14 Days when the contact is to be deleted or now if the contact is already deleted

I think you forgot this whole part. Needs to be in preFrom.

Reply:
I don't think we need the default, as we always get it from the corresponding External Event (PeerToBeDeleted and PeerDeleted). Or is there any other place where this default would be actually used ?

Reply:
Backwards compatibility.